### PR TITLE
Fix docs for `offset` in `get_histories` and add test

### PR DESCRIPTION
**Pull request by bernt-matthias**
Followup on https://github.com/galaxyproject/bioblend/pull/484/ ping @cat-bro

### CODE REVIEW

**Comment by cat-bro** [comment]
Looks good @bernt-matthias. It improves the description of offset.

File: bioblend/galaxy/histories/__init__.py [hunk 1]
```diff
@@ -218,8 +218,8 @@ def get_histories(
         :param limit: How many items to return (upper bound).
 
         :type offset: int
-        :param offset: skip the first ( offset - 1 ) items and begin returning
-          at the Nth item.
+        :param offset: skip the first (offset) items and begin returning
+          at item at index offset (i.e. start with the element offset+1).
```
Comment (nsoranzo) [inline bioblend/galaxy/histories/__init__.py:222]:
```suggestion
          items at index offset (i.e. start with the element offset+1).
```

I think?

File: bioblend/_tests/TestGalaxyHistories.py [hunk 2]
```diff
@@ -70,6 +70,15 @@ def test_get_histories(self):
         all_histories = self.gi.histories.get_histories()
         assert len(all_histories) > 0
 
+        # Test limit and offset
+        first = self.gi.histories.get_histories(limit=1)
+        others = self.gi.histories.get_histories(offset=1)
+        assert len(others) > 0  # guess the test makes more sense with at least 2 histories
```
Comment (nsoranzo) [inline bioblend/_tests/TestGalaxyHistories.py:76]: Better not to make tests dependent on the order in which they are executed.

```suggestion
        assert len(first) == 1
```

**Review by nsoranzo — APPROVED** [review]
LGTM, let me know when you're ready to move it out of draft.

**Comment by bernt-matthias** [comment]
> ready to move it out of draft.

I just forgot that it was in draft :)

**Pull request merged**